.NET/Microsoft.Recognizers.Text.Number/Models/AbstractNumberModel.cs

```diff
@@ -60,7 +60,7 @@ public List<ModelResult> Parse(string query)
                 // Only support "subtype" for English for now
                 // As some languages like German, we miss handling some subtypes between "decimal" and "integer"
                 if (!string.IsNullOrEmpty(o.Type) &&
-                    Constants.ValidSubTypes.Contains(o.Type) && extractorType.Contains(Constants.ENGLISH))
+                    Constants.ValidSubTypes.Contains(o.Type) && (extractorType.Contains(Constants.ENGLISH) | extractorType.Contains(Constants.SWEDISH)))

```

Comment:
It would be best to declare a method to check for supported cultures, instead of just adding new ``ìf``` conditions.

Reply:
Yes probably. Or, maybe move the responsability for this to the extractor e.g: `extractorType.Supports("subtype")`, or something

Reply:
@tellarin what do you recommend; should I try and add the function in the  extractor, or, should the function be implemented directly in the AbstractNumberModel class? I'm not that sure of your design philosophy yet, so, your advice is welcome.

Reply:
I'll remove this change from this PR since it's not relevant in this context.